INSTALL

```diff
@@ -151,16 +151,16 @@ See also CODE OPTIMIZATION below.
 
 After compilation, the resulting code is stored in the directory
     ./dist.$ARCH
-and the scripts (grass79, ...) in
+and the scripts (grass80, ...) in
```

Comment:
What about dropping the 80 part at least from the documentation? On a reasonable system, it should install as `grass` anyway. I also noticed I usually write `ls` and not `ls830`, `gdalinfo`, not `gdalinfo32`... Seems like less work for the releases, too.

Reply:
Generally fine for me but then we may also need to actually generate a `grass` script at compile time.

Reply:
Yes, fine for me.

> > This PR is meant for testing (do not merge into master!) as it contains needed changes for a GRASS GIS 8.0.0alpha release.
> 
> I don't understand where this will go.

As I understand there are still different opinions (https://lists.osgeo.org/pipermail/grass-dev/2021-May/095136.html and related) on when to create a branch.

>  If it will be released, it should go to the main repo. A branch would make sense to me.

We could asap, as I don't see much point in keeping "7.9" forever:
- merge this PR into the main branch (changing VERSION from "8.0.0alpha" to "8.0.dev")
- then create the "releasebranch_8_0" branch off the main branch (changing VERSION therein to "8.0.0alpha")
- cherry-pick future relevant changes from main branch to "releasebranch_8_0" as we have done with other release branches

Reply:
> Generally fine for me but then we may also need to actually generate a `grass` script at compile time.

A version independend startup script is nice if this does not interfere with installing different GRASS versions in parallel.
So we might need to keep `grass78` and `grass80` anyway? Just to make sure different version in parallel are considered...